content/en/smart-contracts/fundamentals/aggregators

```diff
@@ -0,0 +1,81 @@
+
+1. Client submits subpiece data to an off-chain API. The API generates the subpiece metadata and informs an Aggregator (e.g. subpiece CID, known as pCID, and URL to download the CAR file). 
+
+2. The aggregator hosts an off-chain aggregation node, which downloads these subpiece CAR files and aggregates them into a larger CAR file. 
+
+3. Simultaneously, the aggregator aggregates indexed data segments (based on specs [here](https://github.com/filecoin-project/FIPs/discussions/512)). It runs the PoDSI proofing library and generates proofs for each subpiece pCID and stores it in an off-chain database. 
+
+4. The aggregator uses the [Boost API](https://boost.filecoin.io/experimental-features/fvm-contract-deals) (programmatic) or uses Lotus node (manual) to make storage deals with storage providers for the aggregated larger CAR file. 
+
+5. Storage Providers download the aggregated CAR file and publish storage deals. 
+
+6. Clients can query a proofing endpoint provided by the aggregator (example [here](https://docs.lighthouse.storage/lighthouse-1/filecoin-virtual-machine/podsi-and-deal-info), which will look up the subpiece CID (pCID) in the database and return the PoDSI proof, aggregated CID, and associated dealID.
+
+7. Clients can use the subpiece pCID for on-chain verification with the aggregation smart contract, which will verify merkle proof to ensure the pCID hash matches the commPa of the associated dealID. 
+
+### On-chain aggregation and dealmaking
+
+1. Client submits a subpiece CID (CommPc) with metadata (e.g. metadata of the subpiece and URL to download the CAR file) directly to the aggregation smart contract.
+
+2. The aggregator watches the aggregation contract and when there are enough pieces to produce a 32GiB aggregated piece CID (CommPa), it downloads all subpieces for the aggregated piece from the CAR file URL. 
+
+3. The aggregator aggregates indexed data segments into a larger data file for dealmaking (based on specs [here](https://github.com/filecoin-project/FIPs/discussions/512)).
+
+4. The aggregator combines the subpiece CIDs into the aggregated 32GiB CommP (CommPa) by computing within the aggregation smart contract.
+
+5. The aggregator uses the [Boost API](https://boost.filecoin.io/experimental-features/fvm-contract-deals) to make storage deals with storage providers for the aggregated larger CAR file. 
+
+6. Storage Providers download the aggregated CAR file and publish storage deals. Upon client's request, they can find the data via subpiece CID.
+
+6. Clients can query the aggregation smart contract, which notifies the aggregator to look up the subpiece CID (pCID) in its aggregation node's database and return the PoDSI proof, aggregated CID, and associated dealID.
+
+7. Simultaenously, clients can use the subpiece pCID for on-chain verification with the aggregation smart contract, which will verify merkle proof to ensure the pCID hash matches the commPa of the associated dealID. 
+
```

Comment:
Adding workflow diagram for on-chain aggregation

<img width="1625" alt="shapes 8" src="https://github.com/filecoin-project/filecoin-docs/assets/113331491/56740073-3f1b-41b5-b979-556a79eafc90">